frontend/src/components/layout/ListCards.js

```diff
@@ -7,15 +8,15 @@ const ListCards = ({tickets}) => {
     return a.priority - b.priority;
   });
     return (
-      <>
+      <StyledSection>
         <Card index="example" priority="1">
           <p>Example</p>
           <p>What is time?</p>
         </Card>
         {sortedTickets.length > 0 &&
-          sortedTickets.map((ticket) => {
+          sortedTickets.map((ticket, idx) => {
             return (
-                <Link to={`tickets/${ticket._id}`}>
+                <Link key={ticket._id + idx} to={`tickets/${ticket._id}`}>
```

Comment:
Why would you need the `index` if you already have the `id`? The `id` is always a unique value.

Reply:
Oh! Because if I don't have a unique key on the link AND the card I get an error, I used the id for the card, so I did this on the link... this was a way to make it unique and it got rid of my error :P Am I crazy?